### PR TITLE
Update ha-chart-base.ts: fix "Tooltip: a long name may overflow"

### PR DESCRIPTION
**Pull request by ildar170975**
## Proposed change

If an entity has a long name - currently the name may overflow if does not contain whitespaces:

изображение

изображение

Proposed changes:
-- break a line for a long text;
-- (unrelated issue) make a tooltip a bit wider - to fit long dates like "December 31, 2023 at XX:XX:XX" (cannot test all languages...)

изображение

изображение

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #https://github.com/home-assistant/frontend/issues/18261
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by ildar170975** [comment]
Also, fixed a style for "li" element used to display a state.
https://github.com/home-assistant/frontend/pull/18849/commits/e6d27beda43bb12ce025d25906cfa619a8d99ee7

Before (long state with whitespaces & w/o them):
изображение

After:
изображение

File: src/components/chart/ha-chart-base.ts [hunk 1]
```diff
@@ -450,7 +450,7 @@ export class HaChartBase extends LitElement {
         border-radius: 4px;
         pointer-events: none;
         z-index: 1000;
-        width: 200px;
+        width: 230px;
```
Comment (bramkragten) [inline src/components/chart/ha-chart-base.ts:453]: Why was the width changed? Now the positioning logic will not be correct anymore.

Comment (ildar170975) [reply src/components/chart/ha-chart-base.ts]: I see, you mean "_handleTooltip()" probably. My fault. Thought about increasing the width a little.
Trying to solve it here:
```
        clamp(
          context.tooltip.caretX,
          100,
          this.clientWidth - 100 - this._paddingYAxisInternal
        ) -
        100 +
        "px",
```
with no success so far...
The easiest thing I can do is to restore `200px`, but do not like it.

Comment (ildar170975) [reply src/components/chart/ha-chart-base.ts]: Restored width to 200px. Will try to sort it out later then(((. Sorry, still doing 1st steps.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**